WcaOnRails/app/views/static_pages/teams_committees.html.erb

```diff
@@ -13,7 +13,13 @@
     <% @officers.each do |user, positions| %>
       <div class="badge team-member-badge officer-badge">
         <%= render "shared/user_avatar", user: user %>
-        <div class="team-member-name"><%= user.name %>
+        <% if user.wca_id %>
+          <%= link_to person_path(user.wca_id), title: t("about.structure.users.profile", user_name: user.name) do %>
+            <div class="team-member-name"><%= user.name %>
```

Comment:
Could you please make sure the `div` tags are closed in this PR?
(there are a bunch of them)

Reply:
The div was closed  after the subtext span, it still is but done in a different way, more on the comment below

Reply:
Oh but the fact that the opening div is nested in a `link_to` would have messed up the tag hierarchy (ie: having something like `<a><div></a></div>` which is not valid.
Won't be an issue with the discussion below :)